src/backend/distributed/commands/trigger.c

```diff
@@ -99,6 +101,18 @@ GetExplicitTriggerCommandList(Oid relationId)
 		createTriggerCommandList = lappend(
 			createTriggerCommandList,
 			makeTableDDLCommandString(createTriggerCommand));
+
+		/*
+		 * Appends the commands for the trigger settings that are not
+		 * covered by CREATE TRIGGER command.
+		 */
+
+		if (TriggerIsDisabled(triggerId))
+		{
+			createTriggerCommandList = lappend(
```

Comment:
Maybe more readable to do this
`char * disableTriggerCmd = GetDisableTriggerCommand(triggerId)` then append `disableTriggerCmd` variable to the list.